src/timeseries.js

```diff
@@ -113,7 +113,15 @@ TimeSeries.prototype.getHits = function(key, gran, count, callback) {
     }
 
     for(var ts=from, i=0, data=[]; ts<=to; ts+=properties.duration, i+=1) {
-      data.push([ts, results[i] ? parseInt(results[i], 10) : 0]);
+      var result = results[i];
+      if (result instanceof Array) {
```

Comment:
I would comment here explaining that different clients libs return different things ... just explain why you're doing what you're doing.